chain/chunks/src/lib.rs

```diff
@@ -1757,6 +1779,7 @@ mod test {
             Some("test".parse().unwrap()),
             runtime_adapter,
             network_adapter.clone(),
+            [3; 32],
```

Comment:
maybe create a constant  'TEST_SEED' - and use it in all these tests (rather than passing a 'cryptic' numeric value)